src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -24,6 +26,20 @@ def import_from_package
 
         redirect_to kiwi_image_repositories_path(package.kiwi_image)
       end
+
+      def outdated
```

Comment:
This could be changed to `def show` and then if we need to render more field from the image then we can add it to the json response.